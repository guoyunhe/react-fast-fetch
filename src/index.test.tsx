import { render, screen } from '@testing-library/react';
import { ReactFetch } from '.';

describe('<ReactFetch/>', () => {
  it('render', async () => {
    render(<ReactFetch>Hello</ReactFetch>);

    const elem = await screen.findByText('Hello');

    expect(elem.className).toBe('ReactFetch');
  });
});
