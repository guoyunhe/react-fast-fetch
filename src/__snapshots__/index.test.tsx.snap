// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`members > should match snapshot 1`] = `
{
  "CacheStore": [Function],
  "FetchConfigProvider": [Function],
  "IndexedDBStore": [Function],
  "MemoryStore": [Function],
  "StorageStore": [Function],
  "useFetch": [Function],
  "useFetchConfig": [Function],
}
`;
